fdbclient/Schemas.cpp

```diff
@@ -786,7 +786,8 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
          "coordinators":[
             {
                "reachable":true,
-               "address":"127.0.0.1:4701"
+               "address":"127.0.0.1:4701",
+               "protocol": 1142507688261910528
```

Comment:
I realized that some json implementations might not be able to represent protocol versions exactly. I prefer making this a hex-encoded string